server/config/defaults.cfg

```diff
@@ -8,3 +8,5 @@ progress_file: /tmp/progress.json
 
 [ardana]
 url: http://localhost:9085
+
+[UI_URLs]
```

Comment:
``UI_URLs`` is an icky name for a section name. All of the other section names and header names are in lowercase. It would be a lot nicer to incorporate the ardana url together into a ``[urls]`` section:
```
[urls]
ardana: http://localhost:9085
opsconsole: https://whatever
horizon: https://whatever
```

Reply:
I think we need to talk about this a bit more.  The original purpose of this section was for externalized URLs on the last success page of the UI.  We might add UIs for other things in the future like Kibana, etc.  ardana is not an external URL for the UI to point to.

Reply:
All URLs point to external systems, so I still thing it makes sense to combine them, even if some are used for different purposes. But I really don't want that to stand in the way.